tests/Filter/StringFilterTest.php

```diff
@@ -195,4 +194,22 @@ public function caseSensitiveDataProvider(): array
 
         ];
     }
+
+    /**
+     * @group legacy
+     *
+     * @expectedDeprecation The format option is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.
```

Comment:
```suggestion
     * @expectedDeprecation The "format" option is deprecated since sonata-project/doctrine-orm-admin-bundle 3.x and will be removed in version 4.0.
```